File: app/src/main/java/com/OdiousPanda/rainbow/DataModel/Quote.java
package com.OdiousPanda.rainbow.DataModel;

public class Quote {
    private String main;
    private String sub;
    private String att;

    public String getAtt() {
        return att;
    }

    public void setAtt(String att) {
        this.att = att;
    }

    public String getMain() {
        return main;
    }

    public void setMain(String main) {
        this.main = main;
    }

    public String getSub() {
        return sub;
    }

    public void setSub(String sub) {
        this.sub = sub;
    }

    public void setDefaultQuote() {
        this.main = "If you see these lines.";
        this.sub = "Something went wrong... :|";
    }
}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/Quote.kt
package com.odiousPanda.rainbowKt.model.dataSource

class Quote(var main: String = "", var sub: String = "", var att: String = "") {
    init {
        setDefaultQuote()
    }

    fun setDefaultQuote() {
        main = "Hmmmmm..."
        sub = ""
        att = "*"
    }
}